apertium-eng-spa.eng-spa.lrx

```diff
@@ -8715,7 +8715,185 @@ convertirse en un personaje de moda o importante. -->
           <match lemma="save" tags="vblex.*"><select lemma="ahorrar" tags="vblex.*"/></match>
           <match lemma="money" tags="n.*"/>
         </rule>
-
-
+	
+    <!--
+      eng: as part of
+      spa: como parte de
+    -->
+    <rule weight="1.0">
+      <match lemma="as" tags="*">
+        <select lemma="como" tags="cnjadv"/>
```

Comment:
Is this the correct tag for "como" when it introduces a noun phrase?
Wouldn't it be como.pr?

Reply:
I'm sorry, I don't completely understand. What do you exactly mean?

Reply:
When introducing a noun phrase, "como" is a preposition rather than an adverbial conjunction. 
However, this does not appear as a translation of "as" in the bilingual dictionary, as far as I know. Should it be added? Do we need to add items in "select" to the dictionaries? (I am not too familiar with lexical selection software in Apertium). It does not make sense before a noun phrase (it would be OK for a verb phrase, such as "as follows")

Reply:
English tags "as part of" as `^as<cnjadv>$ ^part<n><sg>$ ^of<pr>$` which then comes out of biltrans as `^as<cnjadv>/como<cnjadv>/cuando<cnjadv>$ ^part<n><sg>/parte<n><f><sg>/papel<n><f><sg>$ ^of<pr>/de<pr>$` at which point lexical selection must choose either `como<cnjadv>` or `cuando<cnjadv>`, so yes, changes would need to be made elsewhere to get a preposition here.

Reply:
But, what preposition could we substitute it with?

Reply:
I think the answer to your question is that Mikel is saying it should be `^como<pr>$` rather than `^como<cnjadv>$`, but that isn't in the dictionary and the English tagger doesn't mark "as" as a preposition in this case.

Reply:
Exactly.